<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthConnectTab</class>
 <widget class="QWidget" name="AuthConnectTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Available Devices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="lstDevices"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnInfo">
          <property name="text">
           <string>Get Info</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/assets/images/button-icons/info.png</normaloff>:/assets/images/button-icons/info.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/assets/images/button-icons/connect.png</normaloff>:/assets/images/button-icons/connect.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Device Visibility</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnBroadcast">
        <property name="text">
         <string>Broadcast</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/assets/images/button-icons/broadcast.png</normaloff>:/assets/images/button-icons/broadcast.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblInfo">
        <property name="text">
         <string>Press the button to become discoverable.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
